config/permission.php

```diff
@@ -77,4 +77,13 @@
      */
 
     'cache_expiration_time' => 60 * 24,
+
+    /*
+     * By default the RoleMiddleware and PermissionMiddleware will trigger an abort with
+     * http 403 status code if the user doesn't have the specified permission or role.
+     *
+     * To override the default 403 response, provide a named route here.
+     */
+
+    'unauthorized_route_name_redirect' => null,
```

Comment:
Can we make this into a config value for an url instead of a route name? I'd suggest `redirect_unauthorized_users_to_url`.

Reply:
@AlexVanderbist, we can, but it is easy to retrive a `url` with a `route name` because it is already decleared on `route/web.php`, to make easy manage url's.

what you think?. but it is ok to set as what you want. :+1:

Reply:
Not everyone is using named routes. Defining this as an url gives us a bit more freedom as well, e.g. passing route params(not sure why you would need this but still)